style.css

```diff
@@ -106,12 +106,13 @@
     left: 0;
     background: #6070ff;
     width: 100vw;
-    height: 100vh;
+    height: 35rem;
```

Comment:
- [ ] You have made a brilliant view experience with the scintillatingly blurred background :smile: but changing the `height` from `100vh` to `35rem` doesn't seem a good move. How about leaving it at `100vh` so that all the user sees is just the menu that clicked on, before deciding on his choice of content to navigate to? This will further enhance the user experience on your site. See how it looks now:

![image](https://user-images.githubusercontent.com/99715640/223505053-a366254a-4ff6-4995-b4aa-82ad932149a4.png)